CHANGELOG.md

```diff
@@ -4,16 +4,22 @@ Changelog for NeoFS Contract
 ## [Unreleased]
 
 ### Added
+- EACL validation in container.setEACL (#330)
+- Contract storage model documentation (#320)
```

Comment:
how do you define the capitalization of the first word? (my vote is for a capital one, but at least they should be synced)

Reply:
Looks nice here.

Reply:
what does "Contract" mean here? why is it "Contract" but the others are "ability", "bump" and so on?

Reply:
Added... some kind of documentation.